src/main/resources/graphql/schema.graphqls

```diff
@@ -46,7 +47,8 @@ type Mutation {
     createMerchant(passwordHash: String!, name: String!, description: String, defaultCategoryId: Int): MerchantPayload!
     updateMerchant(passwordHash: String!, id:Int!, name: String!, description: String, defaultCategoryId: Int): MerchantPayload!
     deleteMerchant(passwordHash:String!, id:Int!): DeleteMerchantPayload!
-    createExpense(passwordHash: String!, title: String!, description: String, amount: Float!): CreateExpensePayload!
+    createExpense(passwordHash: String!, title: String!, description: String, amount: Float!, epochDate: Int, categoryId: Int, merchantId: Int): ExpensePayload!
+    deleteExpense(passwordHash: String!, title: String!, description: String, amount: Float!, epochDate: Int, categoryId: Int, merchantId: Int): ExpensePayload!
```

Comment:
you shouldn't need all of this to delete an expense. You should only need passwordHash and id